backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/inventory/InventoryLineHU.java

```diff
@@ -213,4 +219,35 @@ public HuId getHuIdNotNull()
 
 		return huId;
 	}
+
+	public static InventoryLineHU add(@NonNull final InventoryLineHU hu1, @NonNull final InventoryLineHU hu2)
+	{
+		Check.assumeEquals(hu1.getHuIdNotNull(), hu2.getHuIdNotNull());
+		if (hu1.getId() != null && hu2.getId() != null)
+		{
+			throw new AdempiereException("Can't add InventoryLineHU that's already been persisted");
+		}
+
+		Check.assumeEquals(hu1.getInventoryType(), hu2.getInventoryType());
+
+		final InventoryLineHUBuilder builder = builder()
+				.huId(hu1.getHuIdNotNull())
+				.id(coalesce(hu1.getId(), hu2.getId()));
+
+		if (InventoryType.PHYSICAL == hu1.getInventoryType())
+		{
+			Check.assumeEquals(hu1.getQtyBook().getUomId(), hu2.getQtyBook().getUomId());
+			Check.assumeEquals(hu1.getQtyCount().getUomId(), hu2.getQtyCount().getUomId());
```

Comment:
note that you could also acomodate for different UOMs (if they have a conversion-rate) using the utility-class `de.metas.quantity.Quantitys`...maybe better do that?